kaiko/src/config.ts

```diff
@@ -0,0 +1,6 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'example'
```

Comment:
This is currently an unused variable as there is only one endpoint.

I would instead make every EA similar to the `example` adapter's  `adapter.ts` code with the switch statement routing the logic, even if there is only one endpoint. It makes it a lot easier to extend in the future.